lib/agent/providers.js

```diff
@@ -65,10 +65,22 @@ var map = exports.map = function(cb){
 
 };
 
-var get = exports.get = function(name, cb) {
-
-  var args = arguments,
-      callback = cb || function(){ /* noop */ };
+var get = exports.get = function(name, extra) {
+
+  var cb,
+      options,
+      args = arguments,
```

Comment:
I would suggest, for consistency and styling, to either order the variables alphabetically or put value variables at the beginning (or at the end).